proto/common.proto

```diff
@@ -68,10 +68,18 @@ message Eos {
 
 message TapEvent {
   TcpAddress source = 1;
+  EndpointMeta source_meta = 5;
 
   TcpAddress   destination      = 2;
   EndpointMeta destination_meta = 4;
 
+  ProxyDirection proxy_direction = 6;
+  enum ProxyDirection {
+    UNKNOWN = 0;
```

Comment:
Note: UNKNOWN is used as the default so that we don't misinterpret events where this field is unset.